workshops/views.py

```diff
@@ -521,6 +520,67 @@ def person_password(request, person_id):
     })
 
 
+@login_required
+def person_merge(request):
+    '''Merge information from one Person into another (in case of duplicates).'''
+
+    if request.method == 'POST':
+        form = PersonMergeForm(request.POST)
+        if form.is_valid():
+            # FIXME: what should be serialized?
+            request.session['person_from'] = form.fields['person_from']
+            request.session['person_to'] = form.fields['person_to']
```

Comment:
Right - I figured I'd have to store the Person ID's, but was too sleepy 
to figure out cleaned_data.  Note: you prefer object.pk to object.id ?